applications/OptimizationApplication/python_scripts/controls/shape/shape_control_new.py

```diff
@@ -0,0 +1,88 @@
+import KratosMultiphysics as Kratos
+import KratosMultiphysics.OptimizationApplication as KratosOA
```

Comment:
I was thinking, this won't be a base a base class for `ShapeControl`. Because, as you mentioned, filtering techniques and whatnot will be different for different controls. So we don't need a base class, We can implement different shape controls with the abstract `Control` base class. If we see code duplication, we can move the duplicated code to a central place in shape control folder.